.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{

```

Comment:
what exactly is this? 🤔

Reply:
Try to remove this line and check if the on save behavior still working properly